src/modules/users/controllers/users.controller.ts

```diff
@@ -56,4 +59,18 @@ export class UsersController {
   async fetchOrgUsers(@Param('organization_name') organization: string) {
     return await this.usersService.fetchOrgUserList(organization);
   }
+
+  @Get('/fabricToken')
```

Comment:
fabric-token?

Reply:
I meant its fabricToken, and should be fabric-token as it's route path , LOL.
My bad, should have explained it earlier.

Reply:
I've used `fabric_token` to keep the practice consistent. Already using underscore for `update_password`, & `recover_password` APIs

Reply:
ooh okay, but isnt it like a practice globally to use these as fabric-token, update-password?

Reply:
In that case, I'll update all the APIs.